### DISCUSSION TITLE
[2021.08.25] 베스트셀러

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#40] [Algorithm]
[베스트셀러](https://www.acmicpc.net/problem/1302)

**Comment by pyunji** [comment]
[베스트셀러 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_1302.java)

**Comment by hyk339** [comment]
[베스트셀러 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/1075b38bd49b2dbdfb6bd3db5a6744dfbf4e1f3b)

**Comment by chaeyeon4u** [comment]
[베스트셀러](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_1302.java)

**Comment by leechang4** [comment]
[베스트셀러 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/BOJ/BOJ%201302.java)
